app/assets/stylesheets/application.scss

```diff
@@ -30,7 +30,18 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .post-it {
+ background:#fefabc;
+ padding:15px;
+ font-size:15px;
+ color: #000;
+ width:400px;
 
+ box-shadow: 0px 4px 6px #333;
+ -moz-box-shadow: 0px 4px 6px #333;
+ -webkit-box-shadow: 0px 4px 6px #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Avoid vendor prefixes.<br>`0px` should be written without units as `0`